crates/nu-protocol/src/config/reedline.rs

```diff
@@ -14,23 +12,23 @@ pub struct ParsedKeybinding {
 }
 
 /// Definition of a parsed menu from the config object
-#[derive(Serialize, Deserialize, Clone, Debug)]
+#[derive(Clone, Debug, IntoValue, Serialize, Deserialize)]
 pub struct ParsedMenu {
     pub name: Value,
     pub marker: Value,
     pub only_buffer_difference: Value,
     pub style: Value,
-    pub menu_type: Value,
+    pub r#type: Value,
```

Comment:
Screams for a `#[rename]` attribute, but let's not overcomplicate things.

Reply:
#13761 😉